tonic_validate/metrics/answer_consistency_metric.py

```diff
@@ -33,6 +35,7 @@ async def score(
         main_point_list = parse_bullet_list_response(main_points_response)
         main_point_derived_from_context_list = []
         for main_point in main_point_list:
+            self.prompt += f'{statement_derived_from_context_prompt(statement=main_point, context_list=llm_response.llm_context_list)}\n-------------------\n'
```

Comment:
Can we add the prompt up at the top and just pass an empty context list / main point? Because otherwise if someone tries to get the prompt without scoring it won't display the full prompt.

Reply:
So you want this up in the init, correct?